src/AutoRest.CSharp/Mgmt/Output/ResourceContainer.cs

```diff
@@ -36,24 +37,51 @@ public ResourceContainer(OperationGroup operationGroup, BuildContext<MgmtOutputL
             _context = context;
         }
 
-        public IEnumerable<ClientMethod> RemainingMethods => Methods.Where(m => m.RestClientMethod != PutMethod && m.RestClientMethod != ListMethod?.Method);
+        public IEnumerable<ClientMethod> RemainingMethods => Methods.Where(m => m.RestClientMethod != CreateMethod
+        && m.RestClientMethod != GetRestClientMethod(ListMethod) && !ResourceOperationsListMethods.Any(r => GetRestClientMethod(r) == m.RestClientMethod));
 
-        public RestClientMethod? PutMethod => _putMethod ??= GetPutMethod();
+        public IEnumerable<PagingMethod> RemainingPagingMethods => PagingMethods.Where(m => m.Method != CreateMethod
+        && m.Method != GetRestClientMethod(ListMethod) && m.Method != GetRestClientMethod(SubscriptionExtensionsListMethod) && m.Method != GetRestClientMethod(LocationsListMethod) && !ResourceOperationsListMethods.Any(r => GetRestClientMethod(r) == m.Method));
 
-        public PagingMethod? ListMethod => _listMethod ??= FindListPagingMethod();
+        public RestClientMethod? CreateMethod => _createMethod ??= GetCreateMethod();
+
+        public ResourceListMethod? ListMethod => _listMethod ??= FindListMethod();
 
         public override ClientMethod? GetMethod => _getMethod ??= _context.Library.GetResourceOperation(OperationGroup).GetMethod;
 
-        private PagingMethod? FindListPagingMethod()
+        private ResourceListMethod? FindListMethod()
+        {
+            var listMethods = GetListMethods(true, true);
+            foreach (var listMethod in listMethods)
+            {
+                var restClientMethod = GetRestClientMethod(listMethod);
+                var pathSegments = restClientMethod?.Request.PathSegments;
+
+                // resourceGroups scope
+                if (pathSegments.Any(p => p.Value.IsConstant && p.Value.Constant.Value?.ToString() == "/resourceGroups/"))
```

Comment:
not sure if this should be hard coded.

Type 1 was List me at my parent scope.  parent is variable it could be resourcegroup but it could be subscription

We can validate this with a test case that has a subscription based resource, and its subscription level list method should go in the container.

I wonder if we name these slightly differently
ParentListMethod
AncestorListMethod -> this will be multiple since if your parent is RG then you might have 1 sub scope and 1 tenant scope